lib/acts_as_paranoid/validations.rb

```diff
@@ -75,11 +78,18 @@ def validate_each(record, attribute, value)
           end
         end
       end
+
+      class V6 < V5
+      end
     end
 
     module ClassMethods
       def validates_uniqueness_of_without_deleted(*attr_names)
-        validates_with UniquenessWithoutDeletedValidator[ActiveRecord::VERSION::MAJOR], _merge_attributes(attr_names)
+        if ActiveRecord::VERSION::MAJOR == '6'
+
```

Comment:
Looks like unique validations tests should fail here since there code for AR v6 is missing.

Reply:
Looks like `ActiveRecord::VERSION::MAJOR` is an `Int` so the condition is returning false so its always running the old code and passing the tests 🎉 

What was the reason this condition was added?

Reply:
You're right.  This can be reverted.  I had it in here before when I was trying to figure out why I kept getting "6" is not a valid validator, must be "V5" or "V4".  I understand the code a lot better now than I did when I made that commit :)